File: MapEver/src/de/hu_berlin/informatik/spws2014/mapever/SettingsFragment.java
package de.hu_berlin.informatik.spws2014.mapever;

import android.os.Bundle;
import androidx.preference.PreferenceFragmentCompat;

public class SettingsFragment extends PreferenceFragmentCompat {
    @Override
    public void onCreatePreferences(Bundle savedInstanceState, String s) {
        addPreferencesFromResource(R.xml.pref_general);
    }
}

File: MapEver/src/de/hu_berlin/informatik/spws2014/mapever/SettingsFragment.kt
package de.hu_berlin.informatik.spws2014.mapever

import android.os.Bundle
import androidx.preference.PreferenceFragmentCompat

class SettingsFragment : PreferenceFragmentCompat() {
    override fun onCreatePreferences(savedInstanceState: Bundle?, s: String?) {
        addPreferencesFromResource(R.xml.pref_general)
    }
}